Sources/PostgreSQL/ConnectionInfo.swift

```diff
@@ -0,0 +1,29 @@
+#if os(Linux)
+    import CPostgreSQLLinux
+#else
+    import CPostgreSQLMac
+#endif
+
+public enum ConnInfo {
```

Comment:
I'm not opposed to `ConnectionInfo`. Just seemed like a psql thing to call it `conninfo`. I'll change it.